src/fragments/lib/logging/common/change_log_levels/change_log_levels.mdx

```diff
@@ -0,0 +1,121 @@
+## Change Default Log Level
+
+Below is an example of setting the default log level to `WARN`:
```

Comment:
In this section, you will learn about how to configure the logging level in your application when using the Amplify logger. This is helpful to help you determine the logging level that works best for your use cases, and which levels of errors or warnings you want to capture in Amazon Cloudwatch.

Reply:
Added new section description